### ISSUE TITLE
Handle sticky-header bottom edge case

### ISSUE THREAD

**Issue by joshwnj** [joshwnj/marki#11]
- When you're scrolled to the bottom
- if the last heading is within the visible viewport and it does not match the contents of the sticky-header
- hide the sticky header

**Comment by bernardodiasc** [comment]
My suggestion to handle this is to match something around 2/3 of the vertical viewport to make the sticky heading update. This must be confirmed, but when reading, usually our eyes are on the next topic before the header reach the top of the page. Or maybe hide the sticky header a bit earlier and then show again when reach the next topic.

**Comment by joshwnj** [comment]
Yes, I think it would be good to experiment with some things like you're suggesting. It's possible we can find a consistent show/hide behaviour that improves the transitions overall as well as addressing this edge-case naturally.
